MatrixKit/Models/Room/MXKRoomDataSource.m

```diff
@@ -1060,6 +1060,17 @@ - (CGFloat)cellHeightAtIndex:(NSInteger)index withMaximumWidth:(CGFloat)maxWidth
     return 0;
 }
 
+- (void)refreshBubblesCellData
```

Comment:
I don't think "refresh" is the right term here,
I would suggest "flushPrecomputedBubblesCellData"...